client/src/components/UserShowPage.js

```diff
@@ -1,37 +1,110 @@
 import React, { useState } from 'react';
+import Dropzone from "react-dropzone"
 
 const UserShowPage = (props) => {
     const { id } = props.match.params
+    const currentUser = props.currentUser
     const [user, setUser] = useState({
-      id: '', email: '', createdAt: '', updatedAt: '', username: ''
+      id: '', email: '', createdAt: '', updatedAt: '', username: '', profileImage: ''
+    })
+    const [newProfileImage, setNewProfileImage] = useState({
+      image: {}
     })
 
-    const getUser = async () => {
+    const [uploadedImage, setUploadedImage] = useState({
+      preview: ""
+    })
+
+    const handleImageUpload = (acceptedImage) => {
+      setNewProfileImage({
+        ...newProfileImage,
+        image: acceptedImage[0]
+      })
+  
+      setUploadedImage({
+        preview: URL.createObjectURL(acceptedImage[0])
+      })
+    }
+
+      const addProfileImage = async (event) => {
```

Comment:
Generally users will expect to add a profile image when they create their account, rather than creating an account and then having to update the account to add an image. I would encourage you to consider moving the image uploading to the registration form

Reply:
We're going to do this in a different commit as it's going to require quite a bit of reworking.